### ISSUE TITLE
Localised currency formats for pay-what-you-want system

### ISSUE THREAD

**Issue by SubhadeepJasu** [elementary/appcenter#552]
Change the pay button to local currencies other than dollar outside USA. This would help donators to better access the app with respect to their currency  and money value they understand.

**Comment by btkostner** [comment]
This issue is more difficult than it seems. Stripe only supports a select list of currencies. We would need to get that list before we could determinte what currency to show. Since we can't get a conversion rate from Stripe, we would also need to make a hard set list of minimum payment amounts for each currency.

Once those two issues are addressed, then we _should_ be able to take payment in other supported currencies.

**Comment by SubhadeepJasu** [comment]
True. There are about 180 different known currencies in the world. I think this might be of help: https://stripe.com/docs/currencies and  https://stripe.com/docs/currencies/conversions. If some currencies are not convertible via stripe then they may be excluded for a time being until support for that currency is available.
